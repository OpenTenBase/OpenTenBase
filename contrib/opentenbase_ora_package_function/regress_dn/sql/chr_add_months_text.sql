\c opentenbase_ora_package_function_regression_ora
create extension if not exists opentenbase_ora_package_function;
-- Test cases for function chr(text)
select chr('65'::text) from dual;
select chr('65'::varchar2) from dual;
select chr('65'::nvarchar2) from dual;
select chr('65'::char(2)) from dual;
select chr('65'::character(2)) from dual;
select chr('65'::character varying) from dual;
select chr('65'::nclob) from dual;
select chr('65'::clob) from dual;
