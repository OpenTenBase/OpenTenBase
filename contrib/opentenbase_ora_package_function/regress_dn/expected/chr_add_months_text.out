\c opentenbase_ora_package_function_regression_ora
create extension if not exists opentenbase_ora_package_function;
NOTICE:  extension "opentenbase_ora_package_function" already exists, skipping
-- Test cases for function chr(text)
select chr('65'::text) from dual;
 CHR 
-----
 A
(1 row)

select chr('65'::varchar2) from dual;
 CHR 
-----
 A
(1 row)

select chr('65'::nvarchar2) from dual;
 CHR 
-----
 A
(1 row)

select chr('65'::char(2)) from dual;
 CHR 
-----
 A
(1 row)

select chr('65'::character(2)) from dual;
 CHR 
-----
 A
(1 row)

select chr('65'::character varying) from dual;
 CHR 
-----
 A
(1 row)

select chr('65'::nclob) from dual;
 CHR 
-----
 A
(1 row)

select chr('65'::clob) from dual;
 CHR 
-----
 A
(1 row)
